Sources/Private/Model/DotLottie/DotLottieAnimation.swift

```diff
@@ -21,11 +23,16 @@ struct DotLottieAnimation: Codable {
   var direction: Int? = 1
 
   /// mode - "bounce" | "normal"
-  var mode: String? = "normal"
+  var mode: DotLottieAnimationMode? = .normal
 
   /// Loop mode for animation
   var loopMode: LottieLoopMode {
-    mode == "bounce" ? .autoReverse : ((loop ?? false) ? .loop : .playOnce)
+    switch mode {
+    case .bounce:
+      return .autoReverse
+    default:
```

Comment:
Can we switch exhaustively instead:

```suggestion
    case .normal, nil:
```